tests/EventTests/EventBuilderTest.php

```diff
@@ -24,41 +24,69 @@
 use Optimizely\Logger\NoOpLogger;
 use Optimizely\ProjectConfig;
 
-
 class EventBuilderTest extends \PHPUnit_Framework_TestCase
```

Comment:
Please add one more test, copy it from C# 
https://github.com/msohailhussain/csharp-sdk/blob/phase2_ev_ns_fb/OptimizelySDK.Tests/EventTests/EventBuilderTest.cs

Reply:
I have accomodated numeric conversion test in 
✏️ testCreateConversionEventWithAttributesWithValue()
✏️ testCreateConversionEventNoAttributesWithValue